tests/unit/test_sql_sensor.py

```diff
@@ -17,8 +17,9 @@ def test_clickhouse_sql_sensor_poke(self, mock_hook):
         mock_get_records.return_value = []
         self.assertFalse(op.poke(None))
 
+        # this behavior was changed in future versions
```

Comment:
And that is why the `allow_null` parameter should not be in current CH Sensor implementation which is 1.10.6-oriented as 1.10.6 [did not have this parameter](https://github.com/apache/airflow/blob/1.10.6/airflow/sensors/sql_sensor.py#L61). The implementation should behave as follows: for version 1.10.6 raise an error on `allow_null` parameter provided and `assertFalse` for this test. And for versions ≥1.10.7 where it [is present](https://github.com/apache/airflow/blob/1.10.6/airflow/sensors/sql_sensor.py#L61) the test should `assertTrue`/`assertFalse` based on the value of `allow_null` parameter.